src/briefcase/commands/base.py

```diff
@@ -977,11 +978,26 @@ def update_cookiecutter_cache(self, template: str, branch="master"):
                     # any partial remnants of this initial clone.
                     # If we're getting a GitError, we know the directory must exist.
                     self.tools.shutil.rmtree(cached_template)
+                    git_fatal_message = re.findall(r"(?<=fatal: ).*?$", e.stderr, re.S)
+                    if git_fatal_message:
+                        # GitError captures stderr with single quotes. Because the regex above
+                        # takes everything after git's "fatal" message, we need to strip that final single quote
+                        hint = git_fatal_message[0].rstrip("'")
+
+                        # git is inconsistent with capitalisation of the first word of the message
+                        # and about periods at the end of the message. Normalise those here to ensure
+                        # a clean and uniform presentation
+                        # Also strip the string of any trailing whitespace before applying the period fix.
+                        hint = f"{hint[0].upper()}{hint[1:]}".strip()
+                        hint += "." if hint[-1] != "." else ""
```

Comment:
Please share any ideas of cleaner ways to do this cleanup, particularly the last two normalisation lines. I wasn't sure how deeply to comment this, so I went with as maximal approach as I could think of without just providing the particular examples. Suggestions for better comments are welcome as well :pray:

Reply:
I think `str.capitalize()` is what you're looking for here. 
```suggestion
                        # GitError captures stderr with single quotes. Because the regex above
                        # takes everything after git's "fatal" message, we need to strip that final single quote.
                        hint = git_fatal_message[0].rstrip("'").strip()

                        # git is inconsistent with capitalisation of the first word of the message
                        # and about periods at the end of the message. 
                        hint = f"{hint.capitalize()}{'' if hint[-1] == '.' else '.'}"
```

Reply:
Nice, thanks! Suggestion applied :+1: